mlrun/__main__.py

```diff
@@ -1000,12 +1000,10 @@ def logs(uid, project, offset, db, watch):
     help="Store the project secrets as k8s secrets",
 )
 @click.option(
-    "--notification",
+    "--notifications",
```

Comment:
non backwards compatibility change. add alias?

Reply:
Added both, supporting notifications and notifications